features/step_definitions/logging.rb

```diff
@@ -326,36 +327,75 @@
   })
 end
 
-Given /^logging channel name is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
-  cb_name = 'logging_channel_name' unless cb_name
-  if env.logging_channel_name.empty?
+# start from logging 5.0, we use our own CI pipeline to trigger automation testing
+# add env vars to set channel and catsrc separately for CLO and EO
+# for example:
+# logging_sub_envs:
```

Comment:
@QiaolingTang  We may pass more parameters(For example: eventrouter image name) into cucushift in future. Could we use  more common name logging_env?

Reply:
OK